easyjoblite/consumers/work_queue_consumer.py

```diff
@@ -67,14 +68,15 @@ def process_message(self, body, message):
 
             message.ack()
 
-            if response.status_code >= 400:
+            if response.status_code != JobResponse.SUCCESS:
                 # todo: we should have booking id here in the log message
                 logger.info("{status}: {resp}".format(status=response.status_code,
                                                       resp=response.message))
 
-                if response.status_code >= 500:
-                    # we have a retry-able failure
-                    self._push_message_to_error_queue(body=body, message=message, job=job)
+                if response.status_code in [JobResponse.RETRYABLE_FAILURE, JobResponse.IGNORE_RESPONSE_AND_RETRY]:
```

Comment:
break these as two different elif blocks with additional parameter in _push_message_to_error_queue as update_count as a parameter
```suggestion
                if response.status_code in [JobResponse.RETRYABLE_FAILURE, JobResponse.IGNORE_RESPONSE_AND_RETRY]:
```